product_status/data/function_deactive_default_product_state_data.xml

```diff
@@ -1,26 +1,29 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <data noupdate="0">
+    <data noupdate="1">
         <function name="write" model="ir.model.data">
-            <value model="ir.model.data" search="[('module', '=', 'product_state')]" />
+            <function name="search" model="ir.model.data">
+                <value eval="[('module', '=', 'product_state')]" />
```

Comment:
this change seems risky: you are going to set noupdate false on ALL records and then setting back to true but w/out discrimination.

IMO this write is desiderable only at first install: we should use an install hook instead.

Reply:
Migration script does not work on my computer. I will try with another PR to see if it works on travis. Meanwhile, condition is updated to just update records in `product.state` table.